generators/app/templates/webpack.config.js

```diff
@@ -23,6 +23,7 @@ if (process.env.INCLUDE_WEBPACK_HTML) {
 
 module.exports = {
   entry: [
+    <%- withFetch ? "'whatwg-fetch'," : '' %>
```

Comment:
Will this create an empty string item when the user doesn't wish to include `whatwg-fetch`?